package com.gestion_taches.demo.core.utilisateur.applications.exception;

public class UtilisateurNotFoundException extends RuntimeException{

    public UtilisateurNotFoundException(String message) {
        super(message);
    }

}
